robotcontainer.py

```diff
@@ -251,6 +252,8 @@ def configureButtonBindings(self):
             MoveBothClimbersToMiddleRungCapturePosition(
                 self.leftClimber, self.rightClimber
             )
+        ).whenPressed(
+            StopMovingParts(self.indexer, self.shooter)
```

Comment:
Sequential command group?
Put the waitcommand in here instead?

Reply:
Wasn't aware we can do that, we can I guess